python/sqlflow_submitter/xgboost/predict.py

```diff
@@ -28,24 +28,33 @@ def pred(datasource,
          hive_location="",
          hdfs_user="",
          hdfs_pass="",
-         pai_table=""):
+         pai_table="",
+         model_params=None,
+         train_params=None):
     # TODO(typhoonzero): support running on PAI without MaxCompute AK/SK connection.
     if not is_pai:
         conn = db.connect_with_data_source(datasource)
     label_name = label_meta["feature_name"]
-
     dpred = xgb_dataset(datasource, 'predict.txt', select, feature_metas,
                         feature_column_names, None, is_pai, pai_table, True)
-
     bst = xgb.Booster({'nthread': 4})  # init model
     bst.load_model("my_model")  # load data
-    print("Start predicting XGBoost model...")
```

Comment:
This line is used to indicate the training is started, so that for jobs that need to wait a long time, the user may know it's still running.